server/src/test/java/io/crate/integrationtests/CopyIntegrationTest.java

```diff
@@ -128,7 +129,7 @@ public void testCopyFromFileWithCSVOption() {
     }
 
     @Test
-    public void testCopyFromFileWithUmlautsWhitespacesAndGlobs() throws Exception {
+    public void testCopyFromFileWithUmlautsWhitespacesAndGlobsWithFileSchemeAndUrlEncoding() throws Exception {
```

Comment:
this passes both on linux and windows because the umlauts and whitespaces are encoded within the testcase